ERCS/erc-7786.md

```diff
@@ -126,57 +124,19 @@ interface IERC7786Receiver {
 }
 ```
 
-A **Destination Gateway** is a contract that implements a protocol to validate messages sent on other chains. The gateway can operate in Active or Passive Mode, depending on whether `executeMessage` is invoked by the gateway itself or some other acocunt.
-
-#### Active Mode
+A **Destination Gateway** is a contract that implements a protocol to validate messages sent on other chains. It MUST implement `IERC7786GatewayDestination`.
```

Comment:
There is no `IERC7786GatewayDestination` interface. There was a passive part of the interface, but there is no active part of the interface, so if we drop the passive mode, this interface disappears completelly

```suggestion
A **Destination Gateway** is a contract that implements a protocol to validate messages sent on other chains.
```